apps/launcher/graphicspage.cpp

```diff
@@ -1,6 +1,6 @@
 #include "graphicspage.hpp"
 
-#include <boost/math/common_factor.hpp>
+#include <boost/integer/common_factor.hpp>
```

Comment:
Looks like this change leads to compilation errors.

Reply:
Yeah, math/common_factor.hpp is deprecated in later versions of boost. I can revert this for now, just wanted to get rid of issue. I'll see if we can find some common ground first. (Hence this PR anyway.)